pkgs/development/python-modules/hg-evolve/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, isPy3k
+}:
+
+buildPythonPackage rec {
+  pname = "hg-evolve";
+  version = "9.2.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "08bjrgxv8zrrz5xxydzkjl4a8cw3g62nmzfnvdzxxcrf1c96qw76";
+  };
+
+  doCheck = false;
```

Comment:
:(

Reply:
actually, agree. @xavierzwirtz I guess this requires mercurial library to perform tests, right?

Reply:
Fixed in #81320.

Reply:
I thought there would be an issue passing in mercurial since its packaged with `buildPythonApplication`, turns out no issue though.

Reply:
Application doesn't "spill" any of the python dependencies into the environment. So you largely just get an "executable". These should be able to compose in a given environment, where as you have to worry about package coherence with `buildPythonPackage`